.github/labeler.yml

```diff
@@ -0,0 +1,16 @@
+proto:
```

Comment:
Could we add a label for the containerfiles (== docker) and name it `container`? The docker-related workflows might also fall under it (less sure about that).